Bangazon-Tinkr/Controllers/UsersController.cs

```diff
@@ -46,6 +46,19 @@ public IActionResult CreateNewUser(User userToAdd)
             }
         }
 
+        //api/User/1/Delete
+        [HttpPost("{userIdToDelete}/Delete")]
```

Comment:
I would recommend to changing it to a "Delete" request, even though it will be an update in the repo.